tensorflow_serving/model_servers/main.cc

```diff
@@ -101,6 +101,17 @@ using tensorflow::serving::PredictionService;
 
 namespace {
 
+void ParseProtoTextFile(const string& file, google::protobuf::Message* message) {
```

Comment:
Do you mean tensorflow::Status?
It seems it doesn't work with QCHECK. Would then TF_CHECK need to be used?

What I've done is change it to return a bool. This way it can be used with QCHECK.
Sorry, I'm new to the code base (and C++ =S)

Should you still prefer tensorflow::Status, would the correct implementation be to check if ParseFromString returns true and return Status::OK() or errors::Unknown(...) or some other error. If so, which error?

Thanks,

Per